src/components/Location.react.js

```diff
@@ -116,7 +116,10 @@ Location.propTypes = {
     id: PropTypes.string.isRequired,
 
     /** pathname in window.location - e.g., "/my/full/pathname" */
-    pathname: PropTypes.string,
+    pathname: PropTypes.oneOfType([
+      PropTypes.string,
+      PropTypes.oneOf([null])
```

Comment:
Added because of [this](https://github.com/plotly/dash-core-components/blob/b7c2e4a3d35b53e0e6bd34f25610319fe50804f1/test/test_integration.py#L700-L710) test, which results in an initial callback setting `pathname=None`. We can change the test instead.